fix: Center auction results rows vertically

MounirDhahri opened the pull request:
The type of this PR is: **Bugfix**

<!-- Bugfix/Feature/Enhancement/Documentation -->

<!-- If applicable, write the Jira ticket number in square brackets e.g. `[CX-434]`
     The Jira integration will turn it into a clickable link for you. -->

This PR resolves [CX-1005]

### Description
- Center auction results rows vertically

<img width="284" alt="Screenshot 2021-02-05 at 16 14 40" src="https://user-images.githubusercontent.com/11945712/107051888-447e0c00-67cd-11eb-9469-8ac51cc9fdf8.png">

<!-- Implementation description -->

### PR Checklist (tick all before merging)

<!-- 💡 This checklist is experimental. MX warmly welcomes any feedback about the list or how it impacts your workflow -->

- [x] I have included screenshots or videos to illustrate my changes, or I have not changed anything that impacts the UI.
- [x] I have added tests for my changes, or my changes don't require testing, or I have included a link to a separate Jira ticket covering the tests.
- [x] I have documented any follow-up work that this PR will require, or it does not require any.
- [x] I have added an app state migration, or my changes do not require one. ([What are migrations?](https://github.com/artsy/eigen/blob/master/docs/adding_state_migrations.md))
- [x] I have added a [CHANGELOG.yml](/CHANGELOG.yml) entry or my changes do not require one.


[CX-434]: https://artsyproduct.atlassian.net/browse/CX-434
[CX-1005]: https://artsyproduct.atlassian.net/browse/CX-1005

samchieng reviewed (dismissed):
This looks better! One thing i noticed is there's slightly more spacing between the divider below the text than above it. Is there a way we can make these more even? This isn't launch blocking – so let me know if you prefer for me to create a separate follow up ticket

MounirDhahri commented:
Thanks for the review @samchieng  I fixed the issue you described and also found out we were using the wrong `lineHeight` for the values and fixed it as well. 
<img width="506" alt="Screenshot 2021-02-09 at 13 56 40" src="https://user-images.githubusercontent.com/11945712/107366628-aa74d700-6ade-11eb-87d4-f215d17d8563.png">

samchieng reviewed (commented):
@MounirDhahri can you confirm the spacing between the font and the dividers is 10px? It looks like 15px, but it might just be my eyes! 🤷‍♀️ it looks even now though!

MounirDhahri commented:
For some reason, `TextInput` is adding extra padding on top and it's hard to get it right. I gave up on using it for inline values. I will use `Text` again and get this done.

Here Is a screenshot of how it is now compared to specs.

<img width="962" alt="Screenshot 2021-02-16 at 13 11 14" src="https://user-images.githubusercontent.com/11945712/108061610-dbae5380-7058-11eb-8a89-8f0ea64a8c2e.png">

samchieng reviewed (commented):
Looks good!!
